backends/ruby-rails/app/domain/staff/repository.rb

```
@@ -8,6 +8,10 @@ module Domain
   module Staff
     # スタッフリポジトリのインターフェースです。
     module Repository
+      # @param cond [Domain::Staff::Condition] 検索条件
+      # @return [Integer] 総件数
+      def count_by_condition(cond)                  = raise NotImplementedError
```

RuboCop (1.87.0)

[fatal] unexpected token tEQL
(Using Ruby 2.7 parser; configure using `TargetRubyVersion` parameter, under `AllCops`)

(Lint/Syntax)